doc/lua_api.txt

```diff
@@ -2171,6 +2171,15 @@ Elements
 * `drawborder`: draw button border or not
 * `pressed texture name` is the filename of an image on pressed state
 
+### `image_button[<X>,<Y>;<W>,<H>;<texture name>;<name>;<label>;<noclip>;<drawborder>;<pressed texture name>;<hovered texture name>]`
```

Comment:
`<hovered texture name>` should go in `style` instead of adding a new part to an element, which is last-resort.

Reply:
I've made the changes you requested. Just to confirm for the future: I assume this means that style properties are the new way to go in general, rather than arguments on the element?

Reply:
Considering @kilbith's use-case, is it worth undoing this change?